templates/zerver/help/about-streams-and-topics.md

```diff
@@ -31,19 +31,26 @@ a stream, users can choose to [unsubscribe](/help/unsubscribe-from-a-stream) fro
 stream.
 
 Users can also customize their stream settings; they can
-[pin](/help/pin-a-stream) or star important streams,
-[change the colors](/help/change-the-color-of-a-stream) of streams, and enable desktop
-notifications or muting for streams in order to create a better Zulip
-environment.
+
+* [pin a stream](/help/pin-a-stream)
+* star important streams
+* [change the colors of streams](/help/change-the-color-of-a-stream)
+* [enable desktop notifications for stream](/help/configure-desktop-notifications)
+* [muting notification for streams](/help/mute-a-stream)
+
+in order to create a better Zulipenvironment.
 
 If enabled by the realm administrators, users can
 [create](/help/create-a-stream) streams and [invite](/help/add-or-invite-someone-to-a-stream)
 other users to a stream.
 
-Only realm administrators can make edits to a stream, such as
-[renaming](/help/rename-a-stream), [deleting](/help/delete-a-stream), changing the
-description of a stream, [removing users](/help/remove-someone-from-a-stream) from a
-stream, or changing the accessibility of a stream.
+Only realm administrators can make edits to a stream, such as:
```

Comment:
> Only realm administrators can make **edits to a stream**

I don't think **edits to a stream** is the perfect way to describe those operations.